### DISCUSSION TITLE
Auto shutdown client

### DISCUSSION THREAD

**Discussion by kadkarol** [epoptes/epoptes#118] [Q&A]
This is my problem.
I prepared a cron job that shuts down the LTSP server every day at 15.00. Works well. I want all client stations to shut down a moment earlier. Can I create a cron job on the server that will shut down all LTSP clients using Epoptes. Or maybe there is another way to do it?
System: Mint (ubuntu bionic)

**Comment by alkisg** [comment]
No, at this point epoptes doesn't have a command line tool.
I'll close this issue as it's a duplicate of #8, we may continue the discussion there if you wish.

**Comment by alkisg** [comment]
Duplicate of #8
